### PR TITLE
Add preview dialog for Community guidelines templates

### PR DESCRIPTION
**Pull request by EmiliaPavlova**
- Add preview dialog for Community guidelines templates

Depends on [#5924](https://github.com/alkem-io/client-web/pull/6150)
Awaits for template tags and guideline references

### CODE REVIEW

File: src/domain/collaboration/memberGuidelines/MemberGuidelinesTemplateLibrary/MemberGuidelinesTemplatePreview.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  };
+}
+
+const MemberGuidelinesTemplatePreview: FC<MemberGuidelinesTemplatePreviewProps> = ({ template }) => {
+  const { t } = useTranslation();
+
+  return (
+    <PageContentBlock>
+      <BlockSectionTitle>
+        {t('components.callout-creation.template-step.post-template-default-description')}
```
Comment (EmiliaPavlova) [inline src/domain/collaboration/memberGuidelines/MemberGuidelinesTemplateLibrary/MemberGuidelinesTemplatePreview.tsx:19]: TODO: update translation

File: src/domain/community/community/CommunityGuidelines/CommunityGuidelines.tsx [hunk 2]
```diff
@@ -98,6 +98,7 @@ const CommunityGuidelines: FC<CommunityGuidelinesProps> = ({ communityId, disabl
   return (
     <Formik initialValues={initialValues} validationSchema={validationSchema} enableReinitialize onSubmit={onSubmit}>
       {({ values, handleSubmit, isValid }) => {
+        console.log(values);
```
Comment (ccanos) [inline src/domain/community/community/CommunityGuidelines/CommunityGuidelines.tsx:101]: please remove this console.log

**Review by ccanos — CHANGES_REQUESTED** [review]

File: src/domain/community/community/CommunityGuidelines/CommunityGuidelines.tsx (hunk 2 above)
Comment (EmiliaPavlova) [reply src/domain/community/community/CommunityGuidelines/CommunityGuidelines.tsx]: Removed

File: src/domain/collaboration/communityGuidelines/CommunityGuidelinesTemplateLibrary/CommunityGuidelinesTemplatePreview.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+const CommunityGuidelinesTemplatePreview: FC<CommunityGuidelinesTemplatePreviewProps> = ({ template }) => {
+  const { t } = useTranslation();
+
+  return (
+    <PageContentBlock>
+      <BlockSectionTitle>{t('community.communityGuidelines.templateTitle')}</BlockSectionTitle>
+      <WrapperMarkdown>{template?.guidelines.profile.displayName ?? ''}</WrapperMarkdown>
+      <BlockSectionTitle>{t('community.communityGuidelines.templateDescription')}</BlockSectionTitle>
+      <WrapperMarkdown>{template?.guidelines.profile.description ?? ''}</WrapperMarkdown>
+      {/* <ProfileReferenceSegment references={mockValues.references} profileId={"data?.profile?.id"} /> */}
```
[the remark this answers is not in the document]
Comment (EmiliaPavlova) [reply src/domain/collaboration/communityGuidelines/CommunityGuidelinesTemplateLibrary/CommunityGuidelinesTemplatePreview.tsx]: The server is not ready, there are no references yet.

**Review by ccanos — APPROVED** [review]

**Pull request merged**